src/components/sidebar/helpers/generate-resources-nav-items.ts

```diff
@@ -27,13 +27,16 @@ const generateCommunityForumNavItem = (productSlug?: ProductSlug) => {
 const generateResourcesNavItems = (productSlug?: ProductSlug) => {
   return [
     { heading: 'Resources' },
-    { title: 'All Tutorials', href: 'https://learn.hashicorp.com/search' },
+    { title: 'All Tutorials', href: 'https://learn.hashicorp.com/' },
     generateCommunityForumNavItem(productSlug),
     {
       title: 'Support',
       href: 'https://www.hashicorp.com/customer-success',
     },
-    { title: 'GitHub', href: 'https://github.com/hashicorp/' },
+    {
+      title: 'GitHub',
+      href: 'https://github.com/search?q=repo%3Ahashicorp%2Fvault+repo%3Ahashicorp%2Fwaypoint+repo%3Ahashicorp%2Fdev-portal&type=Repositories&ref=advsearch&l=&l=',
+    },
```

Comment:
@topfunky we currently have an open ticket for making the GitHub link unique to each product: [Github links in product sidebar should be product specific ](https://app.asana.com/0/1202114367927919/1202445483168886/f). What do you think about reverting the change to this link in favor of this ticket?

Reply:
Reverted in 24dc7f8

Reply:
Thanks @topfunky! @zchsh has that related task assigned to them to complete today. 😊